python/paddle/fluid/layers/nn.py

```diff
@@ -1734,7 +1734,7 @@ def sequence_softmax(input, use_cudnn=False, name=None):
     return softmax_out
 
 
-def softmax(input, use_cudnn=True, name=None):
+def softmax(input, use_cudnn=False, name=None):
```

Comment:
add some comments to explain the difference between true or false?

Reply:
Done